### PR TITLE
Software garden and add figshare DOIs

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
# Software garden and add figshare DOIs

In this PR, software gardening is performed to ensure reproducibility and figshare DOIs per plate were added to the main README and the README was simplified to remove the platemap figures that from my perspective could be a bit distracting.

### CODE REVIEW

File: 0.download_data/scripts/download_plates.py [hunk 1]
```diff
@@ -71,6 +71,14 @@
         # save extracted zip file from figshare download to a folder in the `0.download_data` directory
         "output_dir": pathlib.Path("./Plate_4_zip"),
     },
+    # need to be extracted in a second step
```
Comment (gwaybio) [inline 0.download_data/scripts/download_plates.py]: I'm not sure what this comment means... Will this notebook perform this extraction? Or is the user expecting to perform some manual step? 

Also, the notebook is missing a print statement for plate 5

File: README.md [hunk 2]
```diff
@@ -1,15 +1,32 @@
...
+In this repository, we generate image analysis and image-based profiling pipelines to extract and format single-cell morphological profiles.
 
-We train a machine learning model to predict NF1 genotype within a separate repository called: [NF1_SchwannCell_data_analysis](https://github.com/WayScience/NF1_SchwannCell_data_analysis).
-Please visit the above repository for further information on the generation and validation of the logistic regression model.
+We train a machine learning model to predict NF1 genotype, evaluate, and generate figures within a separate repository called: [NF1_SchwannCell_data_analysis](https://github.com/WayScience/NF1_SchwannCell_data_analysis).
+Please visit the above repository for further information on the generation, validation, and figures from this model for the manuscript.
 
-**Note:** All metadata files are located in the [download data module](./0.download_data/). All larger files, including `SQLite` outputs from CellProfiler and `parquet` processed data file from pycytominer, will need to be downloaded using git LFS after the repo is cloned.
+## Goal
```
Comment (gwaybio) [inline README.md:8]: Please reformat to one sentence per line in this subsection

**Review by gwaybio — APPROVED** [review]
LGTM. I think you need to rerun the downloading notebook to test plate 5.

File: README.md (hunk 2 above)
Comment (jenna-tomkinson) [reply README.md:8]: 👍 Thanks for catching that! It is fixed!

File: 0.download_data/scripts/download_plates.py (hunk 1 above)
Comment (jenna-tomkinson) [reply 0.download_data/scripts/download_plates.py]: I have clarified this, it is a mis-copy and paste. It should say that the extraction is performed as a second step. This is fixed.

**Comment by jenna-tomkinson** [comment]
@gwaybio Testing it now, it took 50 minutes to get all 4 plates downloaded. I am expecting an hour and 20 minutes before it is done. Once it is tested, I will merge this PR. 

Thanks!

**Comment by jenna-tomkinson** [comment]
It has been tested. Plate_4 seemed to have a slight folder bug but given bandwidth, it would take a while to figure out so I am assuming it was a bug from rerunning.

I am merging now, thank you @gwaybio!

**Pull request merged**